hail.md

```diff
@@ -263,7 +263,8 @@ There are 3 categories of machines:
    ```python
    #!/usr/bin/env python3
 
-   import hailtctl.batch as hb
+   import hailtop.batch as hb
+   import urllib.parse
```

Comment:
We generally use `shlex.quote` for command line quoting

```suggestion
   from shlex import quote
```

Reply:
Ah nice, I was guessing a bit there what `quote` was meant to be doing